gemini/sample-apps/customer-search/CymbalBankWebDeployed/public/fixed_deposit.html.hbs

```diff
@@ -0,0 +1,595 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fixed Deposit</title>
+    <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,500,600&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="style/style.css" />
+    <link href='https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css' rel='stylesheet'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script defer type="module" src="scripts/script.js"></script>
+    <script src="https://www.gstatic.com/dialogflow-console/fast/df-messenger/prod/v1/df-messenger.js"></script>
+    <style>
+        df-messenger {
+            z-index: 999;
+            position: fixed;
+            bottom: 16px;
+            right: 16px;
+            --df-messenger-primary-color: #4bbb7d;
+            --df-messenger-chat-window-height: 55vh;
+            --df-messenger-chat-window-width: 50vw;
+        }
+        #csvTable {
+            border-collapse: collapse;
+            width: 100%;
+            overflow-x: auto;
+            font-size: 15px;
+        }
+
+        #csvTable th,
+        #csvTable td {
+            border: 1px solid #dddddd;
+            text-align: left;
+            padding: 8px;
+            /* Allow content to break into multiple lines */
+            word-break: break-all;
+        }
+
+        #csvTable th {
+            background-color: #f2f2f2;
+        }
+    </style>
+</head>
+<script>
+    function submitSearch() {
+        document.getElementById('loader').style.display = 'block';
+
+        var siteSearchUrl = 'https://sites.google.com/corp/search/pranshusingh.altostrat.com/cymbalbank?query=';
```

Comment:
You probably shouldn't be searching a corp Google site for a public application.